recipes/zlib/bld.bat

```diff
@@ -0,0 +1,24 @@
+:: Configure step.
+cmake -G "NMake Makefiles" ^
+      -D CMAKE_BUILD_TYPE=Release ^
+      -D CMAKE_PREFIX_PATH=%LIBRARY_PREFIX% ^
+      - DCMAKE_INSTALL_PREFIX:%LIBRARY_PREFIX% %SRC_DIR% ^
+      %SRC_DIR%
+if errorlevel 1 exit 1
+
+:: Build step.
+cmake --build %SRC_DIR% --target INSTALL --config Release
+if errorlevel 1 exit 1
+
+:: cmake Test.
+ctest
+if errorlevel 1 exit 1
```

Comment:
> Shouldn't need the directory. Also, add `if errorlevel 1 exit 1` on the next line.

Removed the directory and added the exit error. (It is really annoying that we need to tell Windows they he is wrong when exiting.)